plugins/woocommerce/includes/wc-core-functions.php

```diff
@@ -456,6 +456,15 @@ function wc_clear_template_cache() {
 	}
 }
 
+/**
+ * Clear the system status theme info cache.
+ *
+ * @since 9.4.0
```

Comment:
@roykho I just noticed this PR was merged before the 9.3.0 code freeze, so we should probably update this tag from 9.4.0 to 9.3.0? :thinking:

Reply:
Good call, PR incoming.

Reply:
We've decided to just leave it as 9.4.0 and change the milestone to match since this isn't super urgent fix.